testapi.pm

```diff
@@ -99,11 +99,11 @@ sub init {
         $serialdev = "hvc0";
     }
     elsif (check_var('VIRSH_VMM_FAMILY', 'xen') && check_var('VIRSH_VMM_TYPE', 'linux')) {
-        if (check_var('VERSION', '12-SP2')) {
-            $serialdev = "hvc0";
+        if (check_var('VERSION', '12-SP1')) {
+            $serialdev = "xvc0";
         }
         else {
-            $serialdev = "xvc0";
+            $serialdev = "hvc0";
```

Comment:
I can use "check_var('VERSION', '12-SP2') or is_casp" or turn it around - this way it will work also for SP3 + ...seems cleaner